server_test.go

```diff
@@ -66,6 +66,7 @@ func RunLocalUDPServerWithFinChan(laddr string, opts ...func(*Server)) (*Server,
 		return nil, "", nil, err
 	}
 	server := &Server{PacketConn: pc, ReadTimeout: time.Hour, WriteTimeout: time.Hour}
+	server.init()
```

Comment:
`ActivateAndServe` already calls `Server.init` so these don't have to be explicitly added.

Reply:
done